### PR TITLE
Codebase cleanup

### PR DESCRIPTION
**Pull request by MeurillonGuillaume**
Moved database functionality into central repository used by this repo and go-strava-daemon.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,7 +3,7 @@ module go-file-processing-daemon
 go 1.13
 
 require (
-	github.com/bikedataproject/go-bike-data-lib/dbmodel v0.0.0-20200722095709-5e193f85665e
+	github.com/bikedataproject/go-bike-data-lib/dbmodel v0.0.0-20200727162450-a47d3b297b9b
```
Comment (antoinerichez) [inline go.mod:6]: blyat seriously ?

File: go.sum [hunk 2]
```diff
@@ -2,8 +2,8 @@ github.com/BurntSushi/toml v0.3.1 h1:WXkYYl6Yr3qBf1K79EBnL4mak0OimBfB0XUf9Vl28OQ
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
 github.com/OneOfOne/xxhash v1.2.5 h1:zl/OfRA6nftbBK9qTohYBJ5xvw6C/oNKizR7cZGl3cI=
 github.com/OneOfOne/xxhash v1.2.5/go.mod h1:eZbhyaAYD41SGSSsnmcpxVoRiQ/MPUTjUdIIOT9Um7Q=
-github.com/bikedataproject/go-bike-data-lib/dbmodel v0.0.0-20200722095709-5e193f85665e h1:7MaItUZWRTQIXu/zwZv2HzSYSJNvfR7/LZLwF5sEmjA=
-github.com/bikedataproject/go-bike-data-lib/dbmodel v0.0.0-20200722095709-5e193f85665e/go.mod h1:nlG/+rgFk6lx8rN0bfxo2F/hDix8bjdCFSPkqjsKZw4=
+github.com/bikedataproject/go-bike-data-lib/dbmodel v0.0.0-20200727162450-a47d3b297b9b h1:g+zqEaYpgJKUBd2fhtzcGi8PqOqnhA7m2oibnWDkhSg=
```
Comment (antoinerichez) [inline go.sum:5]: Seriously?

**Review by antoinerichez — APPROVED** [review]
Are you serious in my eyes?

**Comment by MeurillonGuillaume** [comment]
> Are you serious in my eyes?

Reduce lines by 140 and decrease stress by 5

**Review by BertSchoovaerts — APPROVED** [review]
very noice

**Pull request merged**